db/migrate/20210409175743_create_products.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProducts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :products do |t|
+      t.uuid :uuid
+      t.string :title
+      t.integer :product_type_id
```

Comment:
Read this and fix your migration https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html#method-i-references

Reply:
Thanks, changed type of field.